dark/templates/dark/teams/user_all.html

```diff
@@ -8,7 +8,6 @@
 {% endblock %}
 
 {% block content %}
-
```

Comment:
Najlepiej nie modyfikować pliki jak nie ma na to potrzeby. Łatwiej to będzie potem migrować. Więc jabym raczej dodał linijkę z powrotem